model/src/cfd.rs

```diff
@@ -960,8 +960,8 @@ impl Cfd {
         let proposal = SettlementProposal {
             order_id: self.id,
             timestamp: Timestamp::now(),
-            taker: *payout.taker_amount(),
-            maker: *payout.maker_amount(),
+            taker: *payout.long_amount(),
+            maker: *payout.short_amount(),
```

Comment:
Well, this is not going to be accurate now is it? :D

Reply:
Hehe it looks wrong, but it is in the context of having called:

https://github.com/itchysats/itchysats/pull/1577/files/e75b61cdf4f0857a389e67e00c415d97bd54615d#diff-f078e5601c18fcb29f773d04d16902354c8f66a57b41aedaea122e11a6f8fd3dR943-R950

So it would actually work here.

In any case, I'm still working on this PR because there are other mistakes.